src/officehours_api/migrations/0015_auto_20200515_0941.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.6 on 2020-05-15 13:41
```

Comment:
Can you squash these two migrations together? That will remove the redundancies (they're mostly the same) and combine them both into a single migration, which is easier to read, easier to reverse in the future, etc. Here's how:

https://docs.djangoproject.com/en/3.0/topics/migrations/#migration-squashing